packages/suite/src/components/suite/modals/ReduxModal/ConfirmAddressModal.tsx

```diff
@@ -22,15 +25,48 @@ export const ConfirmAddressModal = ({ addressPath, value, ...props }: ConfirmAdd
 
     if (!account) return null;
 
-    return (
-        <ConfirmValueModal
-            account={account}
-            heading={
+    const getHeading = () => {
+        const receiveSymbolArray = receiveSymbol ? receiveSymbol.toUpperCase().split('@') : [];
```

Comment:
could you please create util in `cryptoSymbolUtils.ts` (if it does not already exist)

Reply:
Done. I used functions from `cryptoSymbolUtils.ts`.